maxfw/utils/image_utils.py

```diff
@@ -59,8 +59,8 @@ def apply_transforms(self, img):
         """
         # verify whether the Normalize or Standardize transformations are positioned at the end
         encoding = [(isinstance(t, Normalize) or isinstance(t, Standardize)) for t in self.transforms]
-        assert sum(encoding[:-1]) == 0, \
-            'A Standardize or Normalize transformation must be positioned at the end of the pipeline.'
+        if sum(encoding[:-1]) != 0:
+            raise ValueError('A Standardize or Normalize transformation must be positioned at the end of the pipeline.')
```

Comment:
```suggestion
            raise ValueError('A Standardize or Normalize transformation can only be positioned at the end of the pipeline.')
```

Reply:
Maybe another if-clause for `encoding[-1]`?

Reply:
@xuhdev something like this:
`if sum(encoding[:-1]) != 0 or encoding[-1]:`?

Reply:
Yes, but the error message would be hard to write if done in one if-clause

Reply:
Is this being caught here?
https://github.com/IBM/MAX-Framework/blob/master/maxfw/core/utils.py#L38

Reply:
Don't look like the same thing to me, maybe @djalova can confirm?

Reply:
I think it's better to ask @kmh4321

Reply:
@xuhdev maybe for now I can revert 3e404c25ee883a8f3581af97e953eee929efc58d since that change is somewhat unrelated to this PR and open an issue?

Reply:
Sure

Reply:
CC: https://github.com/IBM/MAX-Framework/issues/34